components/hab/habitat/plan.sh

```diff
@@ -53,7 +53,15 @@ do_before() {
 do_prepare() {
   _common_prepare
 
-  export rustc_target="x86_64-unknown-linux-musl"
+  # With the musl target, the ring crate is looking for aarch64-linux-musl-gcc,
+  # but the core/musl package provides musl-gcc. This workaround is necessary until the appropriate changes are made to core/musl for aarch64.
+  if [[ "${pkg_target%%-*}" == "aarch64" ]]; then
```

Comment:
do you think this is something that needs to be changed in the aarch64 core/musl package itself?

Reply:
Yes, I will raise a PR in core-packages as it requires some work. Once those changes are in place, I can update this.